src/views/About/index.js

```diff
@@ -61,7 +62,25 @@ const About = () => {
             </Card>
 
             <Card>
-              <CardHeader className="about-header" title="Origins" />
+              <CardHeader className="about-header" title="origins" />
```

Comment:
Can we capitalize 'origins' for consistency with the other card headers?
```suggestion
              <CardHeader className="about-header" title="Origins" />
```